src/Language/GraphQL/Draft/Parser.hs

```diff
@@ -411,6 +414,20 @@ tok :: AT.Parser a -> AT.Parser a
 tok p = p <* whiteSpace
 {-# INLINE tok #-}
 
+literal :: AT.Parser a -> AT.Parser a
```

Comment:
I think we can add some comment describing why needed this and why we have a separate `tok` function as well. @sordina what do you think?

Reply:
Sounds good!